forum/qa-content/qa-page.js

```diff
@@ -870,3 +876,95 @@ function qa_ajax_error()
     }
 
 } ( document ) );
+
+
+/**
+ *	Add button, which will let users to copy link of topic/comment they are currently reading and share it
+ */
+;( function( document ) {
+
+	'use strict';
+
+	var copyToClipboard = window.initializeCopyToClipboard;
+
+    console.info( 'copyToClipboard: ', copyToClipboard, ' /boolean: ' , !!copyToClipboard );
+
+	if ( !!copyToClipboard ) {
+		var shareMethodPopup = ( function() {

```

Comment:
this function name (`shareMethodPopup`) makes no sense to me. Could you explain or rename it?